cmd/peapod-to-fstree/main.go

```diff
@@ -268,3 +299,50 @@ func migrateConfigToFstree(dstPath, srcPath string) error {
 
 	return nil
 }
+
+func actualContainers(appCfg *config.Config) map[cid.ID]struct{} {
+	wsEndpts := fschainconfig.Endpoints(appCfg)
+	if len(wsEndpts) == 0 {
+		log.Fatal("missing fschain endpoints in config")
+	}
+
+	ctx, cancel := signal.NotifyContext(context.Background(), os.Interrupt, os.Kill)
+	defer cancel()
+
+	pk, err := keys.NewPrivateKey()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	c, err := client.New(pk, client.WithContext(ctx), client.WithAutoFSChainScope(), client.WithEndpoints(wsEndpts))
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer c.Close()
+
+	cnrHash, err := c.NNSContractAddress(client.NNSContainerContractName)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	log.Printf("using container contract hash: %s\n", cnrHash)
+
+	cnrCli, err := container.NewFromMorph(c, cnrHash, 0)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	cnrs, err := cnrCli.List(nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	log.Printf("actual containers number: %d\n", len(cnrs))
+
+	res := make(map[cid.ID]struct{})
```

Comment:
`slices.Contains` would require less code

Reply:
it will be called millions of times. or I did not get you

Reply:
nvm, code is already written and correct